src/app/pages/intro/intro.page.html

```diff
@@ -14,7 +14,7 @@ <h1>
 
         <ion-row class="animation-wrapper">
           <ion-col>
-            <ion-img height="400" [src]="animations.welcome"></ion-img>
+            <ion-img class="anime" height="400" [src]="animations.welcome"></ion-img>
```

Comment:
Do we required 'anime' class here? Cant we directly apply css to ion-img tag? Unless there are some ion-imgs that dont require this behaviour in this page.